README.md

```diff
@@ -65,6 +65,7 @@ A curated list of resources for learning about natural language processing, text
 - [Salmon Run](http://sujitpal.blogspot.com/)
 - [Lekta Blog](https://lekta.ai/blog/)
 - [NLP News](http://newsletter.ruder.io/)
+- [NLP Techniques for Extracting Information](https://www.searchtechnologies.com/blog/natural-language-processing-techniques)
```

Comment:
This looks like a blog post, not a blog itself. Please move down to the appropriate section.